packages/vega-typings/types/runtime/index.d.ts

```diff
@@ -25,10 +25,60 @@ export function timeFormatLocale(definition: object): void;
 // Parser
 export function parse(spec: Spec, config?: Config, opt?: { ast?: boolean }): Runtime;
 
+
+
+type LoaderOptions = {
+  /** Base URL prefix prepended to provided URI */
+  baseUrl: string;
+  /** Allows caller to explicitly set loading mode (local or network request). File mode only applies to server-side rendering. */
+  mode: 'file' | 'http';
+  /** Default protocol for protocol-relative URIs, defaults to HTTP */
+  defaultProtocol: string;
+  /** browser `target` attribute for hyperlinks. Only used when sanitizing URI values for use as hyperlink */
+  target: string;
+  /** browser `rel` attribute for hyperlinks. Only used when sanitizing URI values for use as hyperlink */
+  rel: string;
+  /** http request parameters passed to underlying [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) */
+  http: RequestInit;
+  /**
+   * Specify a [crossOrigin](https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image) attribute for images. Only used when sanitizing URI values with option context: "image"
+   * If this property is defined and maps to a value of `null` or `undefined`, then a `no-cors` fetch will be performed for the `Image`. This property can be used to override Vega's default behavior of using `crossOrigin="anonymous"`, which allows images loaded from a different host to be included in exported visualization images (and thereby avoid "tainted canvas errors"), so long as the server provides permission via proper CORS headers.
+  */
+  crossOrigin: 'anonymous' | 'use-credentials' | '' | undefined;
+};
+
+/**
+* Sanitize accepts all options from Loader constructor
+*/
+type SanitizeOptions =
+  | (Partial<LoaderOptions> & {
+        /**
+         * Describes context in which the URI will be used.
+         */
+        context: 'href' | 'image';
```

Comment:
From code search for how `context: '` gets called in the monorepo, I found just 3 sites for href and image.

https://github.com/vega/vega/blob/0b652cd06d9d02172e0740f0ae4544f863f16aa6/packages/vega-scenegraph/src/Handler.js#L138

https://github.com/vega/vega/blob/0b652cd06d9d02172e0740f0ae4544f863f16aa6/packages/vega-scenegraph/src/ResourceLoader.js#L19

https://github.com/vega/vega/blob/0b652cd06d9d02172e0740f0ae4544f863f16aa6/packages/vega-scenegraph/src/ResourceLoader.js#L35